### PR TITLE
fs tweaks

### PR DESCRIPTION
**Pull request by songgao**
1. Show ... menu for folders too, so we can see metadata;
2. Collapse 'Share to other apps' into main popup menu;
3. When time remaining for transfers is not known, just don't show any
thing, ratehr than show '? s';
4. When we don't know the modified timestamp (e.g. for
public/private/team folders in root, or for a TLF in a TLF list), just
don't show a time, rather than showing the epoch;
5. Make clickable area larger for row action ... button and close button in bare view;
6. Fix download badge position on mobile.

### CODE REVIEW

**Comment by songgao** [comment]
It’s probably fine, but I’m ok with not including it in the release too. @jzila WDYT?

**Comment by chrisnojima** [comment]
ok lets defer

File: shared/fs/popups/row-action-popup-container.js [hunk 1]
```diff
@@ -123,11 +123,17 @@ const getRootMenuItems = (stateProps, dispatchProps) => {
       onClick: () => saveImage(path),
     })
   isMobile &&
+    pathItem.type === 'file' &&
+    // TODO: re-enable this (or refactor) when we have in-app share. This
+    // should probably be split up into separate components anyway.
```
Comment (jzila) [inline shared/fs/popups/row-action-popup-container.js]: I thought we decided to include in-app share items in the root menu:
* Share to other app
* Share to chat conversation

Comment (songgao) [reply shared/fs/popups/row-action-popup-container.js]: Oh; I wasn't aware of that (or I didn't remember). Will clarify in the TODO comment.

**Comment by jzila** [comment]
As @chrisnojima says, let's wait. We're already going through the release checklist.

**Comment by songgao** [comment]
Please take a look at last two commits ("remove second level ... " and "fix download badge ..."). Thanks!

File: shared/actions/fs/common.native.js [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
-}
+export const shareNative = ({payload: {path, routePath}}: FsGen.ShareNativePayload) =>
+  Saga.sequentially([
+    Saga.put(FsGen.createDownload({intent: 'share', path})),
+    getTransferPopupAction(path, routePath),
+  ])
+
+export const saveMedia = ({payload: {path, routePath}}: FsGen.SaveMediaPayload) =>
+  Saga.sequentially([
+    Saga.put(FsGen.createDownload({path, intent: 'camera-roll'})),
```
Comment (jzila) [inline shared/actions/fs/common.native.js]: please keep `intent` and `path` in the same order for the various `FsGen.createDownload` calls.

**Pull request merged**